src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
-body {
+@import 'variables';
+@import 'mixin';
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include center_the_element;
+}
```

Comment:
The BEM naming convention suggests that the block name should be lowercase and describe the purpose of the block, rather than its state or representation. Consider renaming `.page` to a more descriptive class name that follows the BEM naming conventions.